lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -125,6 +125,10 @@ for barclamp in $deactivation_list; do
     continue
   fi
 
+  if [[ $barclamp == "tempest" ]] && [[ $applied_proposals == Usage:* ]]; then
+    continue
+  fi
```

Comment:
Sorry, I don't get the first sentence.  `tempest` needs to be the first element of the `deactivation_list`, or that the `if` of `tempest` needs to be before the previous `test`.